roles/mesos/tasks/follower.yml

```diff
@@ -78,6 +78,15 @@
   tags:
     - mesos
 
+- name: set follower resources
+  sudo: yes
+  copy:
+    dest: /etc/mesos-slave/resources
+    content: "{{ mesos_resources }}"
+  notify: restart mesos follower
```

Comment:
this should remove follower metadata, like this: https://github.com/kenjones-cisco/microservices-infrastructure/blob/feature/slave-resource/roles/mesos/tasks/follower.yml#L95